fusedrug_examples/interaction/drug_target/affinity_prediction/bimodal_mca_PPI/runner.py

```diff
@@ -0,0 +1,220 @@
+'''
+training BimodalMCA affinity predictor (see https://pubs.acs.org/doi/10.1021/acs.molpharmaceut.9b00520)
+with focus on multiprocessing to improve GPU utilization
+'''
+
+import os
+import tensorflow as tf
+import pytorch_lightning as pl
+from pytorch_lightning.callbacks import Callback
+import os
+os.environ['TOKENIZERS_PARALLELISM'] = 'false'
+import hydra
+from utils import hydra_resolvers #examples.fuse_examples.interaction.drug_target.affinity_prediction.bimodal_mca_PPI.
+
+
+#hydra.verbose = True
+#import logging
+#log = logging.getLogger(__name__)
+from omegaconf import DictConfig, OmegaConf
+from hydra.utils import instantiate, call
+import sys
+import socket
+from fuse.utils.file_io import read_simple_int_file
+from clearml import Task
+from pl_model import AffinityPredictorModule  #TODO: fix import path for fuse drug
+from pl_data import AffinityDataModule          #TODO: fix import path
+import colorama
+colorama.init(autoreset=True)
+from colorama import Fore, Back, Style
+
+print('CUDA_VISIBLE_DEVICES=', os.environ['CUDA_VISIBLE_DEVICES'] if 'CUDA_VISIBLE_DEVICES' in os.environ else None)
+
+
+def _instantiate_if_needed(x):
+    if isinstance(x, (dict, DictConfig)):
+        if '_target_' in x:
+            x = instantiate(x)
+    return x
+ 
+
+CONFIGS_DIR = os.path.join(os.path.dirname(__file__), 'configs') 
+SELECTED_CONFIG = 'train_bmmca_full_PPI.yaml'
+
+for name,func in hydra_resolvers.items():
+   OmegaConf.register_new_resolver(name, func)
+
+
+# logging.info(f'CONFIGS_DIR={CONFIGS_DIR}')
+# logging.info(f'SELECTED_CONFIG={SELECTED_CONFIG}')
+
+def session_manager_and_logger_setup(cfg:dict)->None:
```

Comment:
I do wonder what should we do with this, as we've migrated into having session_manager in biofm.